htdocs/conf/conf.php.example

```diff
@@ -37,7 +37,7 @@ $dolibarr_main_url_root='';
 // $dolibarr_main_document_root='/var/www/dolibarr/htdocs';
 // $dolibarr_main_document_root='C:/My web sites/dolibarr/htdocs';
 //
-$dolibarr_main_document_root='';
+$dolibarr_main_document_root=realpath(__DIR__.'/..'); 
```

Comment:
This let think that we can keep value realpath(__DIR__.'/..'); as value.
I am not sure that such value will work with some special setup (like when we use diralias of apache instead of virtual host).

Reply:
I moved these suggestions in the comments, and fixed some spelling and end of line whitespaces.